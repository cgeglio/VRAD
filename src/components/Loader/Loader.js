import React from 'react';
import './Loader.scss';

const Loader = () => {
  return (
    <div className="loader">
      <img src={process.env.PUBLIC_URL + '/noun_Travel_2913599.svg'} alt="travel spinner/" className="loader-img" />
    </div>
  );
};

export default Loader;
