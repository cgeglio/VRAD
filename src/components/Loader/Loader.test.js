import React from 'react';
import { shallow } from 'enzyme';
import Loader from './Loader';

describe('Loader', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
