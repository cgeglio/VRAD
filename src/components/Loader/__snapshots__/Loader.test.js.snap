// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match the snapshot with all data passed in correctly 1`] = `
<div
  className="loader"
>
  <img
    alt="travel spinner/"
    className="loader-img"
    src="/noun_Travel_2913599.svg"
  />
</div>
`;
